src/main/java/com/igot/cb/discussion/service/impl/DiscussionServiceImpl.java

```diff
@@ -110,13 +112,22 @@ public ApiResponse createDiscussion(JsonNode discussionDetails, String token) {
             jsonNodeEntity.setIsActive(true);
             ((ObjectNode) discussionDetails).put(Constants.IS_ACTIVE,true);
             jsonNodeEntity.setData(discussionDetails);
+            long postgresTime = System.currentTimeMillis();
             DiscussionEntity saveJsonEntity = discussionRepository.save(jsonNodeEntity);
+            long postgresTimeTaken = System.currentTimeMillis() - postgresTime;
+            ApiMetricsTracker.recordDbOperation("discussion/create", "postgres","insert", postgresTimeTaken);
```

Comment:
avoid creating variables for one time usage... 
ApiMetricsTracker.recordDbOperation("discussion/create", "postgres", "insert", System.currentTimeMillis() - postgresTime);

do check tracking is enabled or not -- like if (ApiMetricsTracker.isEnabled) then call this method